QUICKSTART.md

```diff
@@ -142,63 +142,50 @@ In order to avoid that node in your allocation, a simple MSR read test can be
 performed to determine whether or not msr-safe is functional.  In order to
 attempt a read of an MSR through msr-safe, the ```geopmread``` command line
 utility can be utilized.  This must be invoked on the compute nodes through
-```aprun```.  The following bash script can be utilized to determine the
-hostnames of the nodes that are malfunctioning:
-
-#### check_rdmsr.sh
-```bash
-#!/bin/bash
-
-# Try to read a MSR.  Print the hostname if it fails.
-geopmread POWER_PACKAGE_TDP package >& /dev/null || hostname | sed -e 's/nid[0]*//'
-
-# Always return true to satisfy set -e
-true
-```
+```aprun```.
 
 In order to generate a comma separated list of bad nodes, the following bash
 function can be utilized:
 
 ```bash
-get_bad_nodes(){
+get_bad_nodes() {
+# Try to read a MSR.  Print the hostname if it fails.
     CHECK_RDMSR=check_rdmsr.sh
-    OUTPUT=$(aprun -n${COBALT_JOBSIZE} -N1 -q ${CHECK_RDMSR})
-    OUTPUT=$(echo -n ${OUTPUT} | sed 's/\>/,/g;s/,$//;s/ //g')
-    echo -n ${OUTPUT}
+    echo '#!/bin/bash' > $CHECK_RDMSR
+    echo "geopmread POWER_PACKAGE_TDP package >& /dev/null || hostname | sed -e 's/nid[0]*//'" >> $CHECK_RDMSR
```

Comment:
needs domain index

Reply:
This is gross.  Why are you inlining the script?

Reply:
With the script inlined, the use can just paste the function into their cobalt script.  Otherwise there is an additional step of creating the script and making sure it is in the working directory.

Thanks for the pointer to the missing domain index.

Reply:
@bgeltz don't yuck someone else's yum!

Reply:
I just don't like the idea of putting all this code here.  IMHO, it was better as a standalone script because you may want to manually aprun it outside of the context of a batch script.